src/app/app.component.ts

```diff
@@ -22,9 +18,9 @@ export class MyApp {
 
     // used for an example of ngFor and navigation
     this.pages = [
-      { title: 'Home', component: HomePage },
-      { title: 'Directory', component: DirectoryPage },
-      { title: 'Login', component: LoginPage }
```

Comment:
With Ionic lazy-loading, you have to import the components using strings instead of the actual component object.